azurerm/internal/services/cosmos/common/indexing_policy.go

```diff
@@ -85,6 +109,43 @@ func flattenCosmosDBIndexingPolicyExcludedPaths(input *[]documentdb.ExcludedPath
 	return excludedPaths
 }
 
+func flattenCosmosDBIndexingPolicyCompositeIndex(input []documentdb.CompositePath) []interface{} {
+	if input == nil {
+		return nil
```

Comment:
this still needs to return an empty list here:

```suggestion
		return []interface{}{}
```